pkg/cluster/meta/topology.go

```diff
@@ -127,6 +127,7 @@ func AllComponentNames() (roles []string) {
 // TiDBSpec represents the TiDB topology specification in topology.yaml
 type TiDBSpec struct {
 	Host            string                 `yaml:"host"`
+	ListenAddress   string                 `yaml:"listen_address,omitempty"`
```

Comment:
I think `listen_address` is a bad name.

Reply:
em... Any suggestion?

Reply:
Maybe call it `listen_host`? (since Address should contain the port part)

Reply:
great,

Reply:
done.